msbuild/Xamarin.iOS.Tasks.Windows/Tasks/UnpackHotRestartFrameworks.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Ionic.Zip;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Mono.Cecil;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Xamarin.iOS.HotRestart.Tasks
+{
+	public class UnpackFrameworks : Task
+	{
+		#region Inputs
+
+		[Required]
+		public ITaskItem[] ReferencedAssemblies { get; set; }
+
+		[Required]
+		public string IntermediateOutputPath { get; set; }
+
+		#endregion
+
+		#region Outputs
+
+		[Output]
+		public ITaskItem[] Frameworks { get; set; }
+
+		#endregion
+
+		public override bool Execute()
+		{
+			var frameworks = new List<ITaskItem>();
+
+			Directory.CreateDirectory(Path.Combine(IntermediateOutputPath, "Frameworks"));
```

Comment:
The source code doesn't quite follow our style guidelines, maybe reformat all the added cs files?

The IDE should pick up and apply our [.editorconfig](https://github.com/xamarin/xamarin-macios/blob/main/.editorconfig), so it shouldn't be a lot of work.

Reply:
Done